### DISCUSSION TITLE
Option to change the image resolution in ggplot_image() ?

### DISCUSSION THREAD

**Discussion by solvi808** [rstudio/gt#1482] [General]
Hello, for my use case i find gt working very well to create large formatted output tables.

But my table happens to have 400+ up to thousands of rows, and each row has a corresponding image. This is becoming a problem for filesize and save times. I only need a dpi of say 50 for my images to be readable, or at least smaller (absolute) image dimensions. Is there any way to achieve this and reduce the absolute image size and filesize ?

I tried changing the `height` parameter in f.ex. `ggplot_image(height = px(img_height), aspect_ratio = 4)` , and from what I can tell it changes nothing except the relation to the aspect ratio? but it does not change the image file size.

**Comment by rich-iannone** [comment]
Hi @solvi808 , this is currently not possible so I'm going to create an issue from this discussion.
